server/api-manager.js

```diff
@@ -15,8 +15,8 @@ const URL_SALE = BASE + "sale";
 const URL_STAFF = BASE + "staff";
 const URL_ENROLLMENT = BASE + "enrollment";
 const URL_APPOINTMENT = BASE + "appointment";
-//const APIKEY = "76af57a017f64fcd9fc16cc5032404a0";
-const APIKEY = "7db287c206374b2f911ddc918879983d"; //dan's API key. Use it if you really need it
+const APIKEY = "76af57a017f64fcd9fc16cc5032404a0";
+//const APIKEY = "7db287c206374b2f911ddc918879983d"; //dan's API key. Use it if you really need it
```

Comment:
This didn't need to be changed. I didn't realize until today, but the 1000 calls/day thing only applies once we leave the Mindbody Sandbox.